src/sfx.cpp

```diff
@@ -564,6 +564,36 @@ void sfx_play_wm_footstep(int player_model_id, int player_walkmap_type)
 	}
 }
 
+void sfx_process_wm_highwind(bool is_highwind_moving)
+{
+	static bool playing = false;
+
+	constexpr int default_sfx_id = 493; // Tiny Bronco SFX
+	constexpr int default_channel = 1;
+
+	if (is_highwind_moving && !playing)
+	{
+		if (use_external_sfx)
+		{
+			char track_name[64];
+			sprintf(track_name, "sfx_highwind");
+			nxAudioEngine.playSFX(track_name, default_sfx_id, default_channel, 0.0f, true);
+		}
+		else
+		{
+			common_externals.play_sfx_effects(64, default_sfx_id, 0, 0, 0);
+		}
+		playing = true;
```

Comment:
This variable should be set manually on the `common_externals.play_sfx_effects` as it doesn't return a value. But on the external case, it should be assigned based on the return value of `nxAudioEngine.playSFX`

Reply:
Done